components/LoginSection/LoginSection.jsx

```
@@ -193,6 +238,35 @@ const LoginSection = () => {
           )}
         </div>
       </form>
+      <Drawer
+        anchor="bottom"
+        open={termsAcceptanceDrawer}
+      >
```

ESLint

[error] Replace `⏎········anchor="bottom"⏎········open={termsAcceptanceDrawer}⏎······` with `·anchor="bottom"·open={termsAcceptanceDrawer}`

(prettier/prettier)